doc/source/ray-observability/ray-distributed-debugger.rst

```diff
@@ -35,7 +35,32 @@ Create a new virtual environment and install dependencies.
 Start a Ray cluster
 ~~~~~~~~~~~~~~~~~~~
 
-Run `ray start --head` to start a Ray cluster.
+.. tab-set::
+
+  .. tab-item:: Local
+
+    Run `ray start --head` to start a local Ray cluster.
+
+  .. tab-item:: KubeRay
+
+    Follow the instructions in :doc:`the RayCluster quickstart <../cluster/kubernetes/getting-started/raycluster-quick-start>` to set up a cluster.
+    You will need to connect VS Code to the cluster, one way to do that is to add
+
+    .. code-block:: yaml
+
+        ports:
+        - containerPort: 22
+          name: ssd
+
+
+    to the `ray-head` container and make sure sshd is running in the `ray-head` container. You can then connect VS Code to the cluster by running
```

Comment:
> make sure sshd is running in the `ray-head` container

The wording may not be very precise. Setting the container port does not start the SSHD service. Instead, the KubeRay operator configures the Kubernetes Service based on the head Pod's port configuration, allowing you to use the Kubernetes Service to access the port instead of directly accessing the head Pod.

ref: https://faun.pub/should-i-configure-the-ports-in-kubernetes-deployment-c6b3817e495

![image](https://github.com/user-attachments/assets/99f04f85-c825-44d3-990b-6619646435cc)

Reply:
Thanks -- I added some more instructions now that should it make very clear for people that they actually need to run sshd in the container :)